cypher-shell/src/main/java/org/neo4j/shell/Main.java

```diff
@@ -75,44 +78,70 @@ void startShell(@Nonnull CliArgs cliArgs) {
         if (cliArgs.getVersion() || cliArgs.getDriverVersion()) {
             return;
         }
-        Logger logger = new AnsiLogger(cliArgs.getDebugMode());
-        PrettyConfig prettyConfig = new PrettyConfig(cliArgs);
+        Logger logger = new AnsiLogger( cliArgs.getDebugMode() );
+        PrettyConfig prettyConfig = new PrettyConfig( cliArgs );
+
+        CypherShell shell = new CypherShell( logger, prettyConfig, ShellRunner.shouldBeInteractive( cliArgs ),
+                cliArgs.getParameters() );
+       System.exit( runShell( cliArgs, shell, logger ) );
```

Comment:
Weird indentation.

Reply:
Can you split this into two lines? I know this is identical, but I think it will be easier to follow.